README.md

```diff
@@ -594,7 +594,10 @@ The mountPath should be changed to `/var/run/cri.sock` and hostPath should be po
 `/var/run/containerd/containerd.sock` for containerd. If using helm chart, the flag `--set cri.hostPath.path=/var/run/containerd/containerd.sock`
 can set the paths for you.
 
-*Note*: When using other container runtime instead of dockershim, make sure also setting kubelet in instances.
+*Note*: 
+
+* When using other container runtime instead of dockershim, make sure also setting kubelet in instances.
+* If you are using EKS provided AMI, there is no need to containerd.sock to cri.sock, as EKS AMI [symlinks containerd.sock to dockershim.sock](https://github.com/awslabs/amazon-eks-ami/blob/2289937fe9fb3933d066b6ae72a4299b2624e154/files/bootstrap.sh#L470). In addition, if planning upgrade to containerd runtime while using EKS provided AMI, please follow the instructions in our documentation.
```

Comment:
should this be "there is no need to mount containerd.sock to cri.sock"

Reply:
While this is accurate, I feel we should encourage users to mount `containerd.sock` explicitly in to `cri.sock` when they are using `containerd`. Using `dockershim.sock` as the CRI socket name for `containerd` cases will be confusing. For ex - We can't really say if they are using `dockershim` or `containerd` as CRI by looking at the VPC CNI spec and creates unnecessary confusion in my opinion.

Reply:
Also we can put a note that pods have to be deleted prior to migration since IPs will leak.

Reply:
Hi @achevuru , Given EKSAMI Symlink containerd.sock to dockershim.sock (https://github.com/awslabs/amazon-eks-ami/blob/2289937fe9fb3933d066b6ae72a4299b2624e154/files/bootstrap.sh#L470) - so, when the customer was running containerd, they were were confused and and MNG team was asserting that it was un-necessary since they symlink in AMI.  I think, this is just to avoid the confusion, until the symlink is removed?

--- 
The other link on document from mng team (which I should add) should explain about how to migrate from one runtime to another.

Reply:
So, we're now referring to the socket name VPC CNI should use. MNG symlink will help in general - I mean if an user has some other application relying on `dockershim.sock` name, migration will be seamless for them. But for VPC CNI, we can encourage users to mount the correct socket under `cri.sock` as a best practice recommendation. Maybe we can retain the existing wording but add an additional sentence stating our recommendation to mount it under cri.sock (?)